test/utils/hmac_validator_test.rb

```diff
@@ -7,39 +7,60 @@ module ShopifyAPITest
   module Utils
     class HmacValidatorTest < Test::Unit::TestCase
       def test_invalid_signature
-        query = ShopifyAPI::Auth::Oauth::AuthQuery.new(
-          code: "somecode",
-          shop: "some-shop.myshopify.com",
-          state: "1234",
-          timestamp: "123456",
-          hmac: "invalid",
-          host: "host"
+        auth_query = ShopifyAPI::Auth::Oauth::AuthQuery.new(
+          code: query[:code],
```

Comment:
Not really a problem since this is just test code, but we're calling the `query` method quite a few times here. I wonder if it would be worth assigning the value to a variable so we're not calling the method several times for each test.

Reply:
Updated!